### ISSUE TITLE
Windows: Creating machine default fails

### ISSUE THREAD

**Issue by marcofranssen** [docker/toolbox#108]
After succesfull installation of `Docker-toolbox1.8.1b.exe` on `Windows 8.1`, I get following output in the `Docker quickstart terminal`

```bash
Creating machine default...
rm: cannot remove '//.docker/machine/machines/default': No such host or network path
Looks like something went wrong... Press any key to continue
```

It seems the path is incorrect, I expected the path to be `~/.docker/machine/machines/default`.

I'm running `Virtualbox 5.0.2 r102096`. Any ideas on how to solve this issue? Can I solve it myself or do I have to wait for new release?

**Comment by mchiang0610** [comment]
@marcofranssen Thanks for reporting this. We are looking into this issue.

**Comment by danrie** [comment]
Similar issue, but not exactly the same.  Using Windows 8.1.  All previous versions of Docker, docker-machine, boot2docker, and even MSYS-Git were removed prior to installation.  Ditto for C:\Users\myUserName\.docker and .boot2docker folders.  (Links to boot2docker.iso removed were from VirtualBox, as was the 'default' VM in VirtualBox 4.3.30 r101610.)  So it was a clean install (which I've repeated several times).
Install of DockerToolbox-1.8.1b completes successfully. The 'default' VM gets rebuilt successfully upon opening the Docker Cli ('default' VM shows up as running in VirtualboxBox console).  Lots of lines of text fly by, but eventually the script gets stuck waiting for a response from the docker server. The failed connection message just repeats forever.  At no time do I see the definition of the environmental variables that provide the server IP and location of CERTS.  (the server IP does show up in often in the text scroll)
Any subsequent docker commands fail due to inability to connect to  the server (times out).  After adding the missing environment variables (probably correct but not certain), docker commands fail due to missing CERTS such as ca.pem, server.pem, or server-key.pem (after adding some certs to .docker/machine/machines/default/certs or a similar path as indicated in the error messages).
Unable to run any docker commands.  "docker run hello-world" fails as do other docker commands.  

Docker was  working prior to removal old versions and the install of the toolbox.  (Now I have no docker functionality at all.)

**Comment by ismail-s** [comment]
@mchiang0610 

I also got an issue trying to create machine default. See this screenshot:
![image](https://cloud.githubusercontent.com/assets/3778413/9523273/147ab200-4cd1-11e5-9729-59163ff04002.png)

A bit of background: C:\Users\Ismail is an old account that no longer works. My current user account is "C:\Users\Ismail S" (yes, with the space, which is what I expect caused the problem).

In start.sh, I commented out the lines:

    $DOCKER_MACHINE rm -f $VM &> /dev/null || :
    rm -rf ~/.docker/machine/machines/$VM

and it worked fine.

As it is an old account, I wasn't too fussed with it trying to trash all of those files, but this is a major, if not critical, issue IMO.
